yotierror/activitydetails.go

```diff
@@ -0,0 +1,14 @@
+package yotierror
+
+import "errors"
+
+var (
+	// InvalidTokenError means that the token used to call GetActivityDetails is invalid. Make sure you are retrieving this token correctly.
+	InvalidTokenError = errors.New("invalid Token")
```

Comment:
Not sure if this was intentional? (`TokenDecryptError` is lowercase _"token"_)
```suggestion
	InvalidTokenError = errors.New("invalid token")
```

Reply:
Ah, no that wasn't intentional, fixed in a46d0ac